css/properties/background-image.json

```diff
@@ -22,7 +22,7 @@
               "notes": "If the <code>browser.display.use_document_colors</code> user preference in <code>about:config</code> is set to <code>false</code>, background images will not be displayed."
             },
             "firefox_android": {
-              "version_added": true
+              "version_added": "4"
```

Comment:
I'm assuming this is based on the value for Firefox. If so we should pull down the note, or substantiate that the behavior is different for Android.

Reply:
The assumption is correct.  Sure thing, pulling the notes down now!